mlir/test/Quantum/AdjointTest.mlir

```diff
@@ -16,7 +16,7 @@
 
 // CHECK-LABEL:      @workflow_plain
 func.func private @workflow_plain() -> tensor<4xcomplex<f64>> attributes {} {
-  %c1_i64 = arith.constant 1 : i64
+  %val = "test.op" () : () -> (i64)
```

Comment:
Does this `"test.op"` have value 1? Or I guess my question is if we are reusing existing tests and overloading them with other purposes, are we sure the changed version and the original have the same functionality?

Reply:
The `%val` is a garbage value. The previous test is purely a syntactic check that does nothing with the meaning that `%c1_i64` is has a statically known value of 1. It only verifies that the value defined as `%c1_i64` is used in the correct locations. Therefore replacing this operation with a garbage value still is a valid test.

If we were to do a more complex transformation that would act on the fact that `c1_i64` is a statically determined value of 1, then `test.op` wouldn't be a suitable replacement.

Reply:
Got it, thanks!